Contents copied from python-agate. See
https://github.com/wireservice/agate/issues/712 for the source and the
rationale.

diff --git a/tests/__init__.py b/tests/__init__.py
index e69de29..ee8beb5 100644
--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -0,0 +1,4 @@
+import locale
+
+# The test fixtures can break if the locale is non-US.
+locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
